libbeat/filter/config.go

```diff
@@ -0,0 +1,14 @@
+package filter
+
+type DropFieldsConfig struct {
+	Fields []string `config:"fields"`
+}
+
+type IncludeFieldsConfig struct {
+	Fields []string `config:"fields"`
+}
+
+type FilterConfig struct {
+	DropFields    *DropFieldsConfig    `config:"drop_fields"`
```

Comment:
I would suggest that we call it `exclude_fields` instead of `drop_fields` as we already have `exclude_lines`.